torch/_inductor/fx_passes/quantization.py

```diff
@@ -167,6 +167,24 @@ def get_dequantize_qconv_pt2e_pattern(users=1):
     KeywordArg("postop_algorithm"),
 )
 
+qlinear_tensor_pt2e_pattern = CallFunction(
```

Comment:
Since most of the code is same as `qlinear_pt2e_pattern`, maybe define a function to get `torch.ops.onednn.qlinear_pointwise.default` or `torch.ops.onednn.qlinear_pointwise.tensor`?

Reply:
Thanks. It's fixed.